### PR TITLE
BaSP-TG-81: Project form.

### PR DESCRIPTION
**Pull request by gregorioelias**
- [x] Add route on Layout Component
- [x] Create Form
- [x] Add feature
- [x] On editing, form should contain previous info
- [x] Edit feature
- [x] Success Message on Add or Edit

### CODE REVIEW

File: src/Components/Projects/AddNew.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      })
+    })
+      .then((response) => response.json())
+      .then((data) => console.log(data))
+      .then(() => alert('Project created succesfully.'))
+      .catch((error) => console.log(error));
+  };
+
+  return (
+    <section className={styles.container}>
```
Comment (cocolobos) [inline src/Components/Projects/AddNew.js:66]: the best approach is to use div

File: src/Components/Projects/AddNew.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            placeholder="Assign the admins"
+            value={project.admins}
+            onChange={handleChange}
+          ></input>
+          <span>Must have less than 50 characters. Only admin names.</span>
+        </div>
+        <div>
+          <input type="submit" name="project-submit" value="ADD NEW PROJECT"></input>
+        </div>
+        <a href="/projects">Cancel</a>
```
Comment (cocolobos) [inline src/Components/Projects/AddNew.js:161]: this should be a button with an onclick function to redirect

File: src/Components/Projects/EditProject.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        endDate: changeDate(endDate),
+        clientName: clientName,
+        active: Boolean.valueOf(active),
+        employees: addMembers(employees),
+        admins: admins
+      })
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
```
Comment (cocolobos) [inline src/Components/Projects/EditProject.js:53]: do we need this console log ?

File: src/Components/Projects/EditProject.js [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        alert('Project updated succesfully!');
+      })
+      .catch((error) => {
+        console.log(error);
```
Comment (cocolobos) [inline src/Components/Projects/EditProject.js:57]: this should be a console.error

File: src/Components/Projects/EditProject.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+              setAdmins(e.target.value);
+            }}
+            value={admins}
+          ></input>
+          <span>Must have less than 50 characters. Only admin names.</span>
+        </div>
+        <div>
+          <input type="submit" name="project-submit" value="EDIT PROJECT"></input>
+        </div>
+        <a href="/projects">Cancel</a>
```
Comment (cocolobos) [inline src/Components/Projects/EditProject.js:171]: this should be a button with an onclick function to redirect

File: src/Components/Projects/index.js [hunk 6]
```diff
@@ -1,11 +1,44 @@
...
+  };
 
-function Projects() {
   return (
     <section className={styles.container}>
-      <h2>Projects</h2>
+      <div>
+        <h2>Projects</h2>
+        <List list={list} deleteItem={deleteItem} setList={setList} />
+        <a href="/projects/add">Add New Project</a>
```
Comment (cocolobos) [inline src/Components/Projects/index.js:38]: this should be a button with an onclick function to redirect

**Review by cocolobos — COMMENTED** [review]
On the form, the admin and employee to be added to project should be selected from a request to API. Talk with me for more details

**Review by dayanagm96 — APPROVED** [review]

**Pull request merged**